OptimoveSDK/optimove-sdk/src/main/java/com/optimove/sdk/optimove_sdk/optipush/events_dispatch_service/NotificationOpenedEventDispatchService.java

```diff
@@ -1,74 +0,0 @@
-package com.optimove.sdk.optimove_sdk.optipush.events_dispatch_service;
-
-
-import android.app.Service;
-import android.content.Intent;
-import android.os.IBinder;
-
-import androidx.annotation.Nullable;
-
-import com.optimove.sdk.optimove_sdk.main.Optimove;
-import com.optimove.sdk.optimove_sdk.main.events.core_events.notification_events.ScheduledNotificationOpenedEvent;
-import com.optimove.sdk.optimove_sdk.main.events.core_events.notification_events.TriggeredNotificationOpenedEvent;
```

Comment:
there is a whole bunch of events in `com.optimove.sdk.optimove_sdk.main.events.core_events.notification_events` not needed anymore

Reply:
No, all push activity is to be handled by the Kumulos side of the SDK so the OptiPush related events can be removed.